cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/rest/CloudControllerRestClientImpl.java

```diff
@@ -149,6 +155,12 @@
     private RestTemplate restTemplate;
     private CloudSpace target;
 
+    // Both of these will be utilized in a future commit:
```

Comment:
If a cherry-pick is needed, is it possible to cherry-pick this commit and not the other and introduce a NPE?